src/main/java/org/folio/marccat/dao/persistence/T_DUAL_REF.java

```diff
@@ -1,15 +0,0 @@
-package org.folio.marccat.dao.persistence;
-
-/**
- * @author paulm
- * @since 1.0
- */
-public class T_DUAL_REF extends T_SINGLE {
-  public static final short YES = 1;
-  public static final short NO = 0;
-
-  public static boolean isDual(short code) {
-    return code == 1;
-  }
-
-}
```

Comment:
Why have you removed this class? We use it for the references in authorities

Reply:
If you use it, I put it back and a T_ that I didn't think would display it. Coverage gave me 0 on this class.

Reply:
Thanks